messages/fr/app.php

```diff
@@ -1,6 +1,6 @@
 <?php
 return [
-  'kickstarter_success' => '<h1 class="alert-heading">L\'installation s\'est terminée avec succès!</h1>
+  'kickstarter_success' => '<h1 class="alert-heading">L\'installation s\'est terminée avec succès !</h1>
```

Comment:
@arollmann Not sure if whitespace is correct before the exclamation mark? Its the only change?

Reply:
Yes, it's a detail indeed.  English and German texts do not put spaces before two periods, semicolon, question mark, and exclamation point; the texts in French, yes (in France and in Belgium). Use may be different in French-speaking Switzerland and Quebec ?

Reply:
Thanks for the explanation!